package.json

```diff
@@ -12,7 +12,7 @@
         "generate:type": "yarn workspace go-web-app generate:type",
         "preview": "yarn workspace go-web-app preview",
         "storybook": "yarn workspace go-ui-storybook storybook",
-        "build-storybook": "yarn workspace go-ui-storybook build-storybook",
+        "build:storybook": "yarn workspace go-ui-storybook build-storybook",
```

Comment:
Does this not affect the github workflow?

Reply:
Nice catch :smile: